src/Controls/StreetControls.js

```diff
@@ -211,7 +233,13 @@ class StreetControls extends FirstPersonControls {
             return;
         }
         this.camera.position.copy(position);
-        this.camera.up.copy(position).normalize();
+
+        if (this.view.tileLayer && this.view.tileLayer.isGlobeLayer) {
+            this.camera.up.copy(position).normalize();
+        } else {
+            this.camera.up.set(0, 0, 1);
```

Comment:
Maybe add a warning here that something else other than a globe view is not fully supported yet ?

Reply:
there's no need because both are supported

Reply:
Yes but why are you moving it to `0, 0, 1` ?

Reply:
for planar the up camera is on Z (altitude)

Reply:
Oh my bad I was reading `this.camera.set(0, 0, 1); ` :sweat_smile: